README.md

```diff
@@ -1 +1,39 @@
-# learning-dqrobotics-with-coppeliasim
\ No newline at end of file
+# CoppeliaSim interface with DQ Robotics: From zero to hero
+
+A crash course on the interface with CoppeliaSim robot simulator.
+
+All lessons are in MATLAB Live Script format, meaning that the coding is explained alongside the mathematical equations that they represent. At the end of each lesson, there is a homework section to help the reader internalize the concepts introduced in the lesson.
+
+## How to use
+
+1. Install [MATLAB](https://www.mathworks.com/products/matlab.html).
+2. Clone the latest version of the [DQ Robotics](https://github.com/dqrobotics/matlab) or [download it as a ZIP package](https://github.com/dqrobotics/matlab/archive/refs/heads/master.zip).
```

Comment:
I'm suggesting the master branch here because of the proposed [tutorial structure](https://github.com/dqrobotics/learning-dqrobotics-with-coppeliasim/discussions/1). If we stick to the release version, the tutorial ends in lesson 3. The current release uses `set_joint_positions` in both [LBR4pVrepRobot](https://github.com/dqrobotics/matlab/blob/release/interfaces/vrep/robots/LBR4pVrepRobot.m) and [YouBotVrepRobo](https://github.com/dqrobotics/matlab/blob/release/interfaces/vrep/robots/YouBotVrepRobot.m). This method only works with a passive joints (or dynamics disabled in newer CoppeliaSim versions). Lesson 4 (dynamic world) requires the modifications I sent in this [pull request](https://github.com/dqrobotics/matlab/pull/108).

Should I change it to the release version or add a footnote instead?